lib/rust/ensogl/core/src/animation/loops.rs

```diff
@@ -324,10 +324,7 @@ fn on_frame_closure(
             .then(move || on_after_animations.emit(time_info))
             .then(move || frame_end.emit(time_info))
             .then(move || on_before_layout.emit(time_info))
-            .then(move || {
-                on_before_rendering.emit(time_info);
-                drop(_profiler);
-            })
```

Comment:
Is it intentional? Just making sure, as I don't know this part, and this change wasn't in PR's description.

Reply:
Thanks--fixed the description.